integration-environment-azure/prereqs.tf

```diff
@@ -62,13 +78,21 @@ output "workspace_name" {
 }
 
 output "gen2_adal_name" {
-  value = azurerm_storage_account.account.name
+  value = azurerm_storage_account.adlsaccount.name
 }
 
 output "location" {
-  value = azurerm_storage_account.account.location
+  value = azurerm_storage_account.adlsaccount.location
 }
 
 output "rg_name" {
   value = azurerm_resource_group.example.name
 }
+
+output "blob_storage_key" {
+  value = azurerm_storage_account.blobaccount.primary_access_key
+}
```

Comment:
This seems to be the storage account key. If we are using travis or other ci tools it will display the outputs during the apply phase. Please add the sensitive attribute so this does not get displayed in the apply phase for running the integration tests. https://www.terraform.io/docs/configuration/outputs.html#sensitive-suppressing-values-in-cli-output

Reply:
Added sensitive to the key output. re: cluster. great idea - we'll give it a think.